tests/client/client.php

```diff
@@ -11,7 +11,7 @@
  */
 spl_autoload_register(function ($class) {
     $prefix = 'TrustedLogin\\';
-    $base_dir = __DIR__ . '/src/';
+    $base_dir = dirname(__DIR__,3) . '/src/';
```

Comment:
@Shelob9 This change to autoloader didn't work: https://github.com/trustedlogin/client/actions/runs/4735207086/jobs/8405122115?pr=119#step:7:20

Reply:
This doesn't work, beacuse `__DIR__` is `/var/www/html/wp-content/plugins/client/` and